package.json

```diff
@@ -28,6 +28,7 @@
   },
   "devDependencies": {
     "@types/cookie": "0.4.0",
+    "@types/express": "^4.17.7",
```

Comment:
Should we add `express` as a peer dependency?

Reply:
I just checked some of the top libraries that use this package, but apparently it doesn't seem to be needed since it's not a plugin package, as far as I understand.

Reply:
Ohh yeah, you are right! We also don't use any `express` functionality, right?

Reply:
Yeah, right